{
    "repository": "hgsigner/stringfy",
    "clone_url": "https://github.com/hgsigner/stringfy.git",
    "timestamp": "2023-05-30T11:18:18.469282Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Go/Test] \ud83d\ude80  Start image=crawlergpt:latest\n[Go/Test]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Go/Test]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Go/Test]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Go/Test]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/c66e4e10-fed3-11ed-a890-af2cc187fc11/hgsigner-stringfy] user=0 workdir=\n[Go/Test]   \u2601  git clone 'https://github.com/actions/setup-go' # ref=v1\n[Go/Test] \u2b50 Run Main Set up Go 1.13\n[Go/Test]   \ud83d\udc33  docker cp src=/home/andre-silva/.cache/act/actions-setup-go@v1/ dst=/var/run/act/actions/actions-setup-go@v1/\n[Go/Test]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /var/run/act/actions/actions-setup-go@v1/] user=0 workdir=\n[Go/Test]   \ud83d\udc33  docker exec cmd=[node /var/run/act/actions/actions-setup-go@v1/lib/setup-go.js] user= workdir=\n[Go/Test]   | OCI runtime exec failed: exec failed: unable to start container process: error executing setns process: exit status 1: unknown\r\n[Go/Test]   \u274c  Failure - Main Set up Go 1.13\n[Go/Test] Error response from daemon: Could not find the file /var/run/act/workflow/pathcmd.txt in container 3b4070998843e38266865460e17df35bfbe5a1a0001955d2dfe50a1ef367dfd2\n[Go/Test] failed to remove container: Error response from daemon: No such container: 3b4070998843e38266865460e17df35bfbe5a1a0001955d2dfe50a1ef367dfd2\n[Go/Test] \ud83c\udfc1  Job failed\n",
    "actions_stderr": "Error: Error occurred running finally: Error occurred running finally: Error response from daemon: remove act-Go-Test-c283b741e700b9905f3e6166510e48538a2ca8d0bff1b4728ac91c69db4b0e25-env: volume is in use - [6640703c42e0c81ed1aee8ec0b940c2a1edb11fc1441a63c7cc00028578100a8] (original error: <nil>) (original error: <nil>)\n"
}